b2share/modules/schemas/root_schemas/root_schema_v0.json

```diff
@@ -5,132 +5,191 @@
                 "type": "array",
-                "items": {"type": "string"},
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "contributor_name": {
+                            "type": "string"
+                        },
+                        "contributor_type": {
+                            "enum": ["ContactPerson", "DataCollector", "DataCurator", "DataManager", "Distributor", "Editor", "HostingInstitution", "Producer", "ProjectLeader", "ProjectManager", "ProjectMember", "RegistrationAgency", "RegistrationAuthority", "RelatedPerson", "Researcher", "ResearchGroup", "RightsHolder", "Sponsor", "Supervisor", "WorkPackageLeader", "Other"]
+                        }
+                    }
+                },
                 "uniqueItems": true
             },
-            "resource_type": {
+            "language": {
+                "title": "Language",
+                "description": "The name of the language the document is written in.",
+                "type": "string",
+                "default": "en"
+            },
+            "resource_types": {
                 "title": "Resource Type",
-                "description": "The type of the resource.",
+                "description": "The type(s) of the resource.",
                 "type": "array",
                 "items": {
-                    "type": "string",
-                    "enum": ["Audiovisual", "Collection", "Dataset", "Event", "Image",
-                        "InteractiveResource", "Model", "PhysicalObject", "Service",
-                        "Software", "Sound", "Text", "Workflow", "Other"]
+                    "type": "object",
+                    "properties": {
+                        "resource_type": {
+                            "type": "string"
+                        },
+                        "resource_type_general": {
+                            "enum": ["Audiovisual", "Collection", "Dataset", "Event", "Image", "InteractiveResource", "Model", "PhysicalObject", "Service", "Software", "Sound", "Text", "Workflow", "Other"]
+                        }
+                    }
                 },
                 "uniqueItems": true
             },
-            "alternate_identifier": {
-                "title": "Alternate identifier.",
+            "alternate_identifiers": {
+                "title": "Alternate identifiers",
                 "description": "Any kind of other reference such as a URN, URI or an ISBN number.",
-                "type": "string"
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "alternate_identifier": {
+                            "type": "string"
+                        },
+                        "alternate_identifier_type": {
```

Comment:
As the [OpenAire guidelines specify](https://guidelines.openaire.eu/en/latest/data/field_alternateidentifier.html#alternateidentifiertype-o): `If AlternateIdentifier is used, alternateIdentifierType is mandatory`
This can be done like this:
```json
{
   "type":"object",
   "properties":{
      "alternate_identifier":{
         "type":"string"
      },
      "alternate_identifier_type":{
         "type":"string"
      }
   },
   "required":[
      "alternate_identifier",
      "alternate_identifier_type"
   ]
}
```